MapboxDirections/MBRouteStep.swift

```diff
@@ -388,17 +388,17 @@ struct Road {
     
     init(name: String?, ref: String?, destination: String?) {
         var codes: [String]?
-        if let names = name, let ref = ref ?? destination {
-            // Mapbox Directions API v5 encodes the ref separately from the name but redundantly includes the ref in the name for backwards compatibility. Remove the ref from the name.
-            let parenthetical = "(\(ref))"
-            if names == ref {
+        if let names = name, let gloss = ref ?? destination {
+            // Mapbox Directions API v5 encodes the ref separately from the name but redundantly includes the ref or destination in the name for backwards compatibility. Remove the ref or destination from the name.
```

Comment:
To clarify: This is not true, the `direction` is never mixed into the `name`, but only the `ref`.